CONTRIBUTING.md

```diff
@@ -41,7 +41,9 @@ If you don't see your idea listed, and you think it can contribute to Komodo, do
 * **If your contribution is minor,** such as a fixing a typo, open a pull request.
 * **If your contribution is major,** such as a new feature or bugfix, start by opening an issue first. That way, other contributors can weigh in on the discussion before you do any work.
 
-## Style guide
+## Code style and standard rules guide
```

Comment:
In general, these changes shouldn't be in this PR 
However those looks like a fixes